app/views/vocab_sheets/_vocab_sheet_item.html.erb

```diff
@@ -1,15 +1,13 @@
-<div class="vocab_sheet_item__column vocab_sheet_item__column--<%=size%>">
+<div class="print_view vocab_sheet_item__column vocab_sheet_item__column--<%=size%>">
```

Comment:
This partial is used in both the print and non print views so this new class should probably get removed and attach any related styles to one of the other existing classes in the print style sheet instead.

Reply:
Since you're creating this class name, you should create it with BEM practice in mind. Example: `vocab-sheet-item--print-view`

Reply:
Thank you. Will refactor once I get my head around all the different styles : }